technical-reports/color/color-type.md

```diff
@@ -29,7 +29,7 @@ For example, initially color tokens may be defined as such:
 
 ```json
 {
-  "Majestic magenta": {
+  "Hot pink": {
     "$type": "color",
     "$value": {
       "$hex": "#ff00ff"
```

Comment:
For this file, we should just define it in the `srgb` space, and we don’t need the  `hex` fallback:

```suggestion
      "colorSpace": "srgb",
      "channels": [1, 0, 1]
```

Here and below they should follow the same format

Reply:
Thanks for the feedback! No changes have been added to this file yet. Once we get alignment on the rest, we can make adjustments throughout here.

Reply:
Just a suggestion: it is easier to review if this PR made everything consistent in all files, since that has to be done one way or another. Otherwise it ends up in a state of confusion not being able to tell where the proposal starts and where it ends. Would it be OK to just make everything a consistent format so it’s clear to reviewers?